TWLight/applications/views.py

```diff
@@ -467,6 +468,7 @@ def dispatch(self, request, *args, **kwargs):
         )
 
     def get_success_url(self):
+        # Translators: This message is shown to users once they've successfully submitted their application for review.
```

Comment:
Does this definitely get picked up, or does it need to be on the line right above `_(`?

Reply:
Nope, it doesn't get picked up – just checked. Ugh, will need to go back and look for these comments that aren't placed immediately above the function call (like [this](https://github.com/WikipediaLibrary/TWLight/blob/master/TWLight/resources/views.py#L56), [this](https://github.com/WikipediaLibrary/TWLight/blob/master/TWLight/resources/views.py#L143) etc.).

Reply:
TIL that translator comments have to be immediately above the _(. Thanks!